app/models/user.rb

```diff
@@ -10,6 +10,10 @@ class User < ApplicationRecord
   validates :email, presence: true
   validates :encrypted_password, presence: true
   validates :PostsCounter, presence: true, numericality: { greater_than_or_equal_to: 0 }
+  def admin?(requested_role)
+    role == requested_role.to_s
+  end
```

Comment:
- [ ] Kudos team! 👍🏽  You created the `user.admin` method inside the user model.
- [ ] [OPTIONAL] Kindly add an empty line between lines 12 and 13 to ensure a clean code and separate the admin method from the previous line.